reana_server/reana_admin/cli.py

```diff
@@ -901,10 +923,10 @@ def check_workflows(
 ) -> None:
     """Check consistency of selected workflow run statuses between database, message queue and Kubernetes."""
     from .check_workflows import (
-        check_workflows,
+        InfoCollectionError,
         check_interactive_sessions,
+        check_workflows,
```

Comment:
We can sync in the team about using common import sorting technique across editors.